kolibri/plugins/management/assets/src/vue/class-enroll-page/index.vue

```diff
@@ -1,30 +1,357 @@
 <template>
 
   <div>
-    <p>enroll for class {{ this.classId}}</p>
+    <div class="top-buttons pure-g">
+
+      <div :class="windowSize.breakpoint > 2 ? 'pure-u-1-2' : 'pure-u-1-1 align-center'">
+        <router-link :to="editClassLink">
+          <icon-button
+            :text="$tr('backToClassDetails')"
+            :primary="false">
+            <mat-svg category="navigation" name="arrow_back"/>
+          </icon-button>
+        </router-link>
+      </div>
+
+      <div :class="windowSize.breakpoint > 2 ? 'pure-u-1-2 align-right' : 'pure-u-1-1 align-center'">
+        <icon-button
+          :text="$tr('enrollSelectedUsers')"
+          :primary="true"
+          @click="openConfirmEnrollmentModal"
+          :disabled="selectedUsers.length === 0">
+          <mat-svg category="navigation" name="check"/>
+        </icon-button>
```

Comment:
seeing a bunch of errors trying to enroll:

![image](https://cloud.githubusercontent.com/assets/2367265/23731113/60ac6184-0420-11e7-93b7-367f787e9d29.png)

also some oddness in the pop-up:

![image](https://cloud.githubusercontent.com/assets/2367265/23731118/6f64d1b6-0420-11e7-9938-23b886ec2e17.png)

'no' is selected by default